site/content/en/docs/contrib/roadmap.en.md

```diff
@@ -10,49 +10,25 @@ This roadmap is a living document outlining the major technical improvements whi
 
 Please send a PR to suggest any improvements to it.
 
-# 2021
+# 2022
 
-## (#1) Inclusive and community-driven
+## (#1) GUI
 
-- [x] Maintainers from 4 countries, 4 companies
-- [x] Enhancements approved by a community-driven process
-- [ ] Installation documentation in 5+ written languages
+- [ ] Be able to start, stop, pause, and delete clusters via a GUI
 
-## (#2) User-friendly
+## (#2) Documentation
 
-- [ ] Automatic installation of hypervisor dependencies
-- [ ] Graphical User Interface
-- [ ] Built-in 3rd Party ecosystem with 50+ entries
+- [ ] Consolidate Kubernetes documentation that references minikube
+- [ ] Delete outdated documentation
+- [ ] Add documentation for new features
 
-## (#3) Support all Kubernetes features
+## (#3) ARM64 Support
 
-- [x] Multi-node
-- [ ] IPv6
-- [ ] Usage documentation for 3 leading CNI providers
-- [ ] Automatically publish conformance test results after a release
+- [ ] Add Linux VM support
+- [ ] Add Mac M1 VM support
 
-## (#4) Cross-platform
+## (#4) Reduce Docker Desktop Depencency	
 
-- [x] VM-free deployment to containers (Docker, Podman)
-- [x] Windows as a first-class citizen
-- [x] WSL2 support (no additional VM required)
-- [ ] Firecracker VM support
-- [x] Generic (SSH) driver support
+## (#5) libmachine Refactor
 
-## (#5) Reliable
-
-- [ ] Resource alerts
-- [ ] Time synchronization on HyperKit
 - [ ] Prototype post-libmachine implementation of minikube
```

Comment:
Will split this up further, we have had the "prototype" item shelved for years.

1. add new driver (with QEMU) to replace HyperKit , primarly for Mac arm64
2. fix the provisioner, remove legacy Swarm and add support for other runtimes

We need to move and cleanup the code as well, but that's not a visible feature.